<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0" >
 <class>vectorDialog</class>
 <widget class="QDialog" name="vectorDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>512</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Vectors</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../ElmerGUI.qrc" >
    <normaloff>:/icons/Mesh3D.png</normaloff>:/icons/Mesh3D.png</iconset>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="controlsGroup" >
     <property name="title" >
      <string>Controls</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="vectorLabel" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Vector:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="vectorCombo" >
          <property name="minimumSize" >
           <size>
            <width>280</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="scaleLabel" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Length:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="scaleSpin" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>9999</number>
          </property>
          <property name="singleStep" >
           <number>1</number>
          </property>
          <property name="value" >
           <number>500</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="qualityLabel" >
          <property name="text" >
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="qualitySpin" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>2</number>
          </property>
          <property name="value" >
           <number>8</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="drawEveryNthLabel" >
          <property name="text" >
           <string>Draw every:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="everyNth" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>99999</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="useClip" >
          <property name="text" >
           <string>Use clip plane</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>121</width>
            <height>24</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="randomMode" >
          <property name="layoutDirection" >
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text" >
           <string>Random selection</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QCheckBox" name="useNormals" >
          <property name="text" >
           <string>Compute  normals</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>101</width>
            <height>24</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="scaleByMagnitude" >
          <property name="layoutDirection" >
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text" >
           <string>Scale by magnitude</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="colorGroup" >
     <property name="title" >
      <string>Color</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="lengthLabel" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Color:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="colorCombo" >
          <property name="minimumSize" >
           <size>
            <width>280</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="minLabel" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="minVal" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="maxLabel" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLineEdit" name="maxVal" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QCheckBox" name="keepLimits" >
          <property name="text" >
           <string>Keep limits</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="thresholdGroup" >
     <property name="title" >
      <string>Threshold</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="thresholdVariableLabel" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text" >
           <string>Variable:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="thresholdCombo" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" >
        <item row="0" column="0" >
         <widget class="QLabel" name="thresholdMinLabel" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Min:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="thresholdMin" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QLabel" name="thresholdMaxlabel" >
          <property name="minimumSize" >
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <property name="text" >
           <string>Max:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" >
         <widget class="QLineEdit" name="thresholdMax" >
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>24</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2" >
         <widget class="QCheckBox" name="useThreshold" >
          <property name="text" >
           <string>Use threshold</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3" >
         <widget class="QCheckBox" name="keepThresholdLimits" >
          <property name="text" >
           <string>Keep limits</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>423</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyButton" >
       <property name="text" >
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>Cancel</string>
       </property>
       <property name="icon" >
        <iconset theme="dialog-close" />
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton" >
       <property name="text" >
        <string>Ok</string>
       </property>
       <property name="icon" >
        <iconset theme="dialog-ok-apply" />
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../ElmerGUI.qrc" />
 </resources>
 <connections/>
</ui>
